airflow/providers/google/cloud/hooks/dataproc.py

```diff
@@ -376,15 +376,25 @@ def diagnose_cluster(
         :type metadata: Sequence[Tuple[str, str]]
         """
         client = self.get_cluster_client(location=region)
-        result = client.diagnose_cluster(
+        operation = client.diagnose_cluster(
             project_id=project_id,
             region=region,
             cluster_name=cluster_name,
             retry=retry,
             timeout=timeout,
             metadata=metadata,
         )
-        return result
+        done = False
+        while not done:
+            time.sleep(3)
+            # The dataproc python client has some problems
+            # TypeError: Could not convert Any to Empty
+            try:
+                done = operation.done()
+            except TypeError:
+                pass
```

Comment:
Due to error reported in https://github.com/googleapis/python-dataproc/issues/51

Reply:
ETA this end of this week
https://github.com/googleapis/python-dataproc/issues/51#issuecomment-657888007

Reply:
Solved